frontend/src/components/WofIcon.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <div>
+        <i :class="icon" :style="`font-size: ${size}px`"></i>
```

Comment:
[This guide](https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended) recommends splitting lines on multiple attributes of an html element. Are we supposed to follow this rule or not?

Reply:
It's a good rule of thumb, but I think we use it for more severe cases